src/main/java/io/peacemakr/crypto/ICrypto.java

```diff
@@ -62,7 +62,25 @@
   byte[] decrypt(byte[] cipherText) throws PeacemakrException;
 
   /**
-   * For visibility or debugging purposes, returns a string whihc identifies which
+   * Sing the message.
```

Comment:
*sign

Also, have to be super clear that the blob contains the originally signed message